economy/models.py

```diff
@@ -118,13 +118,13 @@ def terminate_active_session(cls):
 
     @property
     def total_purchases(self) -> int:
-        return self.purchases.count()
+        return self.product_orders.count()
```

Comment:
The method is called `total_purchases`, but returns the number of `product orders`. Rename the method perhaps?